sdk/eventhubs/azure-messaging-eventhubs/src/private/eventhubs_utilities.hpp

```diff
@@ -265,17 +265,17 @@ namespace Azure { namespace Messaging { namespace EventHubs { namespace _detail
     {
       constexpr const char* packageName = "cpp-azure-messaging-eventhubs-cpp";
 
-      options.Properties.emplace("Product", +packageName);
-      options.Properties.emplace("Version", PackageVersion::ToString());
+      options.Properties.emplace("product", +packageName);
```

Comment:
```suggestion
      options.Properties.emplace("product", packageName);
```

Reply:
I'm trying VERY hard to keep this PR to the absolute minimum necessary to fix the problem.

Do you believe that the unary + symbol introduces a problem?